code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -478,8 +478,8 @@
 
 /obj/item/clothing/suit/space/hardsuit/rd
 	icon_state = "hardsuit-rd"
-	name = "prototype hardsuit"
-	desc = "A prototype suit that protects against hazardous, low pressure environments. Fitted with extensive plating for handling explosives and dangerous research materials."
+	name = "hazardous envoirment suit"
+	desc = "An hardsuit designed after an Old-earth type exosuit, renewed by Nanotrasen."
```

Comment:
"an hardsuit"

Reply:
why not, and also english is not my primary language

Reply:
should be "a hardsuit"

Reply:
fixed.